CHANGELOG.md

```diff
@@ -5,10 +5,11 @@
 - Add jpylyzer 2.1.0 for processing jp2 files.
 - Tool upgrades
   - DROID 6.5.2
-  - ExifTool 12.42
+  - ExifTool 12.50
+  - File 5.43
```

Comment:
Should this version also be updated in [tools.properties](https://github.com/harvard-lts/fits/pull/361/files#diff-2512aa731a74f404b204de83d583972bbd5284531e276a45ab33a502562a6f2dR26)?

Reply:
Which version?

Reply:
Here you note `File 5.43`. In tools.properties ([line:26](https://github.com/harvard-lts/fits/pull/361/files#diff-2512aa731a74f404b204de83d583972bbd5284531e276a45ab33a502562a6f2dR26)), version 5.03 is used.

Reply:
That's for Windows. It's an ancient version, but there isn't anything more recent